snap/broken.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/snapcore/snapd/dirs"
 )
 
+// GuessAppsForBroken guesses what apps and services a broken snap has
+// on the system by searching for matches based on the snap name in
```

Comment:
I think it's `s/on/in/`, but better ask a native speaker, @chipaca?

Reply:
I know it's usually 'on' disk, and slightly more often 'in' the filesystem than 'on' the filesystem.

For apps 'in' vs 'on' the system, I can find both (as in googling). 'on' seems more common. Not sure one is more correct than the other. They seem slightly different views on the situation.

Reply:
Both seem correct to me as well. I'd've gone with 'in' myself, but that's probably more about me than about what's being said.